### PR TITLE
Updating message flow topic to include acknowledgements

### PR DESCRIPTION
**Pull request by emileemooreVT**

### CODE REVIEW

File: modules/ROOT/pages/configure-message-flows.adoc [hunk 1]
```diff
@@ -48,7 +48,22 @@ The endpoint is added to the message flow.
 ... From *Message type*, select the message type.
 ... From schema select *Use standard X12-<version>-<message-type>* or select *Customize* to define a custom schema.
 ... Click *Save*. +
+... From schema select *Use standard X12-<version>-<message-type>* or select *Customize* to define a custom schema.
+... Click *Save*. +
 The message type and its validation settings are added to the message flow.
+. In the *Acknowledgment* section, select the check box next to *Sent 997 functional acknowledgment* if you expect your partner to return functional acknowledgments.
```
Comment (vijayang) [inline modules/ROOT/pages/configure-message-flows.adoc]: Please change as:

In the *Acknowledgment* section, select the check box next to *Send 997 functional acknowledgment* if your partner expects your organization to return functional acknowledgments.

File: modules/ROOT/pages/configure-message-flows.adoc [hunk 2]
```diff
@@ -48,7 +48,22 @@ The endpoint is added to the message flow.
...
+.. Click *Select*.
+.. In the *Select an endpoint: Send acknowledgment to <parter>* window, click *Select* to specify the endpoint from which to receive the acknowledgment transactions from your partner. +
```
Comment (vijayang) [inline modules/ROOT/pages/configure-message-flows.adoc]: Please change as:

In the *Select an endpoint: Send to <parter>* window, click *Select* to specify the endpoint to which to send the acknowledgment transactions to your partner.

File: modules/ROOT/pages/configure-message-flows.adoc [hunk 3]
```diff
@@ -48,7 +48,22 @@ The endpoint is added to the message flow.
...
+.. Alternatively, you can create a new receive from partner endpoint:
```
Comment (vijayang) [inline modules/ROOT/pages/configure-message-flows.adoc]: Please change as:

Alternatively, you can create a new send to partner endpoint:

**Review by vijayang — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/configure-message-flows.adoc (hunk 1 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/configure-message-flows.adoc]: Done!

File: modules/ROOT/pages/configure-message-flows.adoc (hunk 2 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/configure-message-flows.adoc]: Done!

File: modules/ROOT/pages/configure-message-flows.adoc (hunk 3 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/configure-message-flows.adoc]: Done!

**Review by vijayang — DISMISSED** [review]

File: modules/ROOT/pages/configure-message-flows.adoc [hunk 4]
```diff
@@ -48,7 +48,22 @@ The endpoint is added to the message flow.
...
+.. Alternatively, you can create a new send to partner endpoint:
+... Click *New*.
+... From *Owner*, select the endpoint owner.
+... From *Protocol*, select the protocol and configure it accordingly:
+* *AS2*
+* *HTTP*
+* *HTTPS*
+* *SFTP*
+... Click *Save*.
+The acknowledgment is added to the message flow.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/configure-message-flows.adoc]:
```suggestion
... Click *Save*. +
The acknowledgment is added to the message flow.
```

**Review by sarathecoubian — APPROVED** [review]

**Pull request merged**